### PR TITLE
Updates navbar UI and creates variations based on user auth

### LINKED ISSUE #105: Update Navbar
Add logo, adjust CSS, and display user status (logged in or not)

### PR DESCRIPTION
**Pull request by Count-MonteCristo**
Makes a series of improvements to the nav bar, including:

- Adds website logo
- Modifies styling to accommodate the logo
- Creates variation based on whether the user is logged in or not (login will be implemented in a future PR)
- Updates nav links with current routes
- Fixes issue with the sticky header rule. No need for individual pages to add top padding anymore

This PR Closes #105

### CODE REVIEW

File: src/components/elements/NavBar.js [hunk 1]
```diff
@@ -1,33 +1,184 @@
...
 
-import {Nav, Navbar, Container, Button, Form} from 'react-bootstrap';
-import '../pages/css/NavBar.css'
+const NavBar = ({ isLoggedIn }) => {
+  const [showProfileMenu, setShowProfileMenu] = useState(false);
+  const location = useLocation();
 
+  useEffect(() => {
+    const handleScroll = () => {
+      const navBar = document.querySelector(".navbar");
```
Comment (dkprioleau) [inline src/components/elements/NavBar.js:15]: Tip: It's best not to connect to the DOM using react. JSX is prefered

File: src/components/elements/NavBar.js [hunk 2]
```diff
@@ -1,33 +1,184 @@
...
+            <Dropdown.Menu>
+              <Dropdown.Item
+                as={Link}
+                to="/profile"
+              >
+                Profile
+              </Dropdown.Item>
+              <Dropdown.Item
+                as={Link}
+                to="/logout"
```
Comment (dkprioleau) [inline src/components/elements/NavBar.js:110]: this can be a very simple page

**Review by dkprioleau — APPROVED** [review]
LGTM!

File: src/components/elements/NavBar.js (hunk 2 above)
Comment (Count-MonteCristo) [reply src/components/elements/NavBar.js:110]: I have it set up like that right now, but the way I'm thinking of implementing the logout is to have that item be a "button" that, when clicked on, removes the token from local storage and sends the user back to the home page.

**Pull request merged**